dotnet/ComputerVision/RecognizeDomainSpecificContent/Program.cs

```diff
@@ -48,7 +48,8 @@ static async Task RecognizeDomainSpecificContentFromStreamAsync(string imageFile
 
                 // Request headers.
                 client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
-                string uri = endpoint + @"/vision/v2.0/models/{specificDomain}/analyze";
+                string uri = endpoint + $@"/vision/v2.0/models/{specificDomain}/analyze";
```

Comment:
Couple of things:
- You don't need the @ sign in that particular string.
- We should put the {endpoint} variable inside the string as well, so it is all consistent. 

Let's do that for the other places where you build these URIs (not just in this file, but the other Rest samples too).